nevergrad/functions/powersystems/test_core.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+from . import core
+
+
+def test_powersystem() -> None:
+    func = core.PowerSystem()
+    x = 7 * np.random.normal(size=func._dimension)
```

Comment:
you shouldnt need to access `_dimension`, there's a `dimension` attribute instead

Reply:
Yes, removed!